.github/workflows/generate-release-crs.yaml

```diff
@@ -0,0 +1,96 @@
+name: Apply Konflux Override Snapshot and create Release CR PR
+on:
+  workflow_dispatch:
+    inputs:
+      revision:
+        description: "SO branch or revision with the manifests"
+        default: "main"
+        required: true
+      environment:
+        description: "Environment to release to"
+        type: choice
+        options:
+          - stage
+          - prod
+        required: true
+      type:
+        description: "Type to release"
+        type: choice
+        options:
+          - component
+          - FBC
+        required: true
```

Comment:
can't we create both PRs at the same time?

Reply:
Good point. Updated it